api/tests/core/users/test_models.py

```diff
@@ -333,6 +334,28 @@ def test_pro_validation_status(self):
         offerers_factories.UserOffererFactory(user=user, validationStatus=ValidationStatus.PENDING)
         assert user.proValidationStatus == ValidationStatus.VALIDATED
 
+    def test_has_partner_page(self):
+        user = users_factories.UserFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerers_factories.UserOffererFactory(user=user, offerer=offerer)
+
+        # Non-permanent & Non-virtual --> has not a partner page:
+        venue_type_code = offerers_models.VenueTypeCode.CULTURAL_CENTRE
+        offerers_factories.VenueFactory(managingOfferer=offerer, venueTypeCode=venue_type_code, isVirtual=False)
+        assert user.has_partner_page == False
+
+        # Permanent & Virtual --> has not a partner page:
+        venue_type_code = offerers_models.VenueTypeCode.LIBRARY
+        offerers_factories.VenueFactory(
+            managingOfferer=offerer, venueTypeCode=venue_type_code, isVirtual=True, siret=None
+        )
+        assert user.has_partner_page == False
```

Comment:
Tu peux utiliser directement la _factory_ d'un lieu virtuel : 
```suggestion
        offerers_factories.VirtualVenueFactory(managingOfferer=offerer, venueTypeCode=venue_type_code)
        assert user.has_partner_page is False
```